scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/simulation/ResumableSimulationDriver.java

```diff
@@ -53,10 +53,13 @@ public class ResumableSimulationDriver<Model> implements AutoCloseable {
   //List of activities simulated since the last reset
   private final Map<ActivityDirectiveId, ActivityDirective> activitiesInserted = new HashMap<>();
 
+  private int countSimulationRestarts;
```

Comment:
Let's add some comments explaining the purpose of these variables - so that readers know that they do no affect semantics, and are merely for keeping track of metrics.

Reply:
done.